pkg/web/templates/partials/transaction/transaction_list.html

```diff
@@ -1,5 +1,5 @@
-<div>
-  <table class="table">
+<div class="table-container">
+  <table class="table table-sm">
```

Comment:
Other table styles will have to be amended for consistency.

Reply:
Something to create a story for?

Reply:
Yep!